lua/wire/zvm/zvm_core.lua

```diff
@@ -694,7 +692,9 @@ function ZVM:Step(overrideSteps,extraEmitFunction)
   return
 end
 
-
+-- Any library that's needed by the entirety of the ZVM instruction set should go here
+-- Platform dependant libraries(GPU or SPU for example) should be added to the env by the platform
+ZVM.Env = {["math"]=math, ["string"]=string, ["bit"]=bit, ["VM"]=ZVM}
```

Comment:
Could just simplify these to `math = math`? Unless you want it to be visually distinct in that way. Also make sure to fix up style.

Reply:
Did that now, along with the changes jay requested